cmd/kops/get_federation.go

```diff
@@ -82,19 +84,21 @@ func RunGetFederations(context Factory, out io.Writer, options *GetFederationOpt
 			return strings.Join(f.Spec.Members, ",")
 		})
 		return t.Render(federations, out, "NAME", "CONTROLLERS", "MEMBERS")
-	} else if output == OutputYaml {
+
+	case OutputYaml:
```

Comment:
I think that could work. What about the case of OutputTable, since the code differs substantially between cluster, isntancegroup, and federation. Maybe I'm misunderstanding? Either way, we can iterate on this as you said.